### ISSUE TITLE
Cluster creation through menu "fails" on CLOUD_PROVIDER/create/all

### ISSUE THREAD

**Issue by willgraf** [vanvalenlab/kiosk#288]
**Describe the bug**
It seems that somehow the make script is trying to create the cluster twice.  After it creates the cluster in `gke/create/all`, it "fails" with a 409 exception that the cluster already exists.  I then can manually call the rest of the `make create` script and the cluster will work. This is happening in the `stable` branch.

**To Reproduce**
Create a cluster, it will likely "fail", but the cluster will be created in the GKE console.

**Expected behavior**
The make script should continue through the creation steps and have a successful code at the end. It should only tell us the cluster creation failed if it actually does fail. 

**Screenshots**
<img width="1432" alt="Screen Shot 2020-03-09 at 10 45 47 AM" src="https://user-images.githubusercontent.com/7930703/76242008-28f03700-61f3-11ea-8e72-9cd137c84b95.png">

**Comment by dylanbannon** [comment]
It looks like this is a result of changes that were made recently to accommodate integration testing (#268), but idk what's going on...

Looking at the screenshot @willgraf posted, it looks like the offending command is on line 41 of `/build-harness/Makefile`. Here is that command and its context:
```
ifndef TRANSLATE_COLON_NOTATION
%:
        @$(SELF) -s $(subst :,/,$@) TRANSLATE_COLON_NOTATION=false
endif
```

I do not know what this means, or whether we actually need it. Commenting it out allows clusters to deploy, but that is not a sustainable fix, since the `/build-harness` files are pulled in at build time from outside our repo.

@osterman you are our only hope!

**Comment by osterman** [comment]
What happens if you set `TRANSLATE_COLON_NOTATION=false` at the top of your `Makefile?

**Comment by dylanbannon** [comment]
That fixed it. Thanks, @osterman!

I'll push a fix this afternoon.

**Comment by dylanbannon** [comment]
Fixed by #289.
